include/rocksdb/cache_allocator.h

```diff
@@ -16,14 +22,35 @@ class CacheAllocator {
   // Name of the cache allocator, printed in the log
   virtual const char* Name() const = 0;
 
-  // Allocate a block of at least size size
+  // Allocate a block of at least size. Has to be thread-safe.
   virtual void* Allocate(size_t size) = 0;
```

Comment:
Is there any reason it is "void*" rather than "char*"?

Reply:
Again it was introduced in #4437. But since even `malloc` return `void*`, I think it is more natural. It means it don't imply typing. Again I don't think the `CacaheAllocator` interface is specific to block cache.